syncstorage-postgres/src/db/mod.rs

```diff
@@ -161,6 +164,24 @@ impl PgDb {
             })
             .collect()
     }
+
+    async fn erect_tombstone(&mut self, user_id: i64) -> DbResult<()> {
+        let naive_datetime = DateTime::from_timestamp(self.timestamp().as_i64() / 1000, 0)
```

Comment:
these should move to the more accurate `as_naive_datetime` from the other PR when that lands